renderer/index.js

```diff
@@ -258,6 +261,22 @@ function jumpToTime (time) {
   }
 }
 
+function changeVolume (delta) {
+  // change volume with delta value
+  setVolume(state.video.volume + delta)
+}
+
+function setVolume (volume) {
+  // check if its in [0.0 - 1.0] range
+  volume = Math.max(0, Math.min(1, volume))
+  if (Cast.isCasting()) {
+    Cast.setVolume(volume)
+  } else {
+    state.video.setVolume = volume
```

Comment:
Probably want to call this `state.video.volume`

Also, it looks like there are two separate volumes now: one for local playback and another for remote (cast) playback. I think that's fine